contrib/docker/Makefile

```diff
@@ -25,7 +25,7 @@
 # Override with make PARALLEL="-j <num_parallel_processes>" where
 #    <num_parallel_processes> = the number of make processes to run in parallel
 # Turn off with make PARALLEL=
```

Comment:
As you have PARALLEL implemented now, this comment is incorrect.  If you specify "PARALLEL=" with no number, then you will either get "-j 22" (because PARALLEL is empty and the build-and-test script will set PARALLEL to the default 22) or you will get "-j" (if the build-and-test script does not detect PARALLEL as being empty ... which I don't think will happen, but bash variable expansion sometimes fools me).  So the comment is wrong.  To use single-threaded parallel (i.e. "-j 1"), you need to use "PARALLEL=1".

Reply:
This change looks good